deplist_test.go

```diff
@@ -141,79 +144,125 @@ func BuildWant() []Dependency {
 	}
 
 	rubySet := []string{
-		"fluent-plugin-kafka",
```

Comment:
nitpick (for previous committer): shame these weren't sorted so that it was easier to see the changes

Reply:
yeah, I basically replaced the ruby 2 project with a ruby 3 one. They are both based on fluentd, but this one is quite a bit different.